src/main/resources/config/application.yml

```diff
@@ -223,6 +223,7 @@ application:
     reval:
       exchange: ${REVAL_RABBIT_EXCHANGE:reval.exchange}
       routingKey.currentpm.update: ${REVAL_RABBIT_CURRENTPMUPDATE_ROUTING_KEY:reval.currentpm.update}
+      routingKey.currentplacement.update: ${REVAL_RABBIT_CURRENTPLACEMENTUPDATE_ROUTING_KEY:reval.currentplacement.update}
```

Comment:
I couldn't start up the service locally. Got an error: `Could not resolve placeholder 'application.rabbit.reval.routingKey.trainee.update' in value "${application.rabbit.reval.routingKey.trainee.update}"`